### PR TITLE
upgrade android api to 28

### PR DESCRIPTION
**Pull request by namrataukirde**
### Ticket Link: 

https://jira.crossroads.org.hk/browse/GCW-2854

### What does this PR do?

It upgrades the android version to point API level 28. 

### Why ? 
Starting November 1, 2019, updates to apps and games on Google Play will be required to target Android 9 (API level 28) or higher. After this date, the Play Console will prevent you from submitting new APKs with a targetSdkVersion less than 28.

NOTE:  Circle ci 2 provides environment variable `CIRCLE_ARTIFACT` which we used previously and it used to point to the path where apk used to save on CI. 

`CIRCLE_ARTIFACT` path = /home/circleci/code/cordova/platforms/android/build/outputs/apk/*.apk

Path where APK gets created after android 7 = /home/circleci/code/cordova/platforms/android/app/build/outputs/apk/debug/*.apk

Thus I have replaced `CIRCLE_ARTIFACT` with new required path.

https://forum.ionicframework.com/t/build-apk-successful-but-no-apk-found-and-build-directory-did-not-found-too/126921

### How I verified android api version is upgraded?

I have used apk analyzer inside an android studio.

Screen Shot 2019-11-04 at 10 23 35 PM

android:targetSdkVersion: An integer designating the API Level that the application targets.

https://developer.android.com/guide/topics/manifest/uses-sdk-element

Please review.

Thanks.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -185,12 +185,14 @@ jobs:
             yarn run azure-filestore download -d google-play -f $GOOGLE_PLAY_KEY_FILE
             mv $GOOGLE_PLAY_KEY_FILE ~/code/fastlane
           working_directory: ~/code
+      - run: yes | sdkmanager --licenses || exit 0
+      - run: yes | sdkmanager --update || exit 0
```
Comment (patrixr) [inline .circleci/config.yml:189]: `exit 1`  otherwise it'll get treated as a success :)

File: fastlane/Fastfile [hunk 2]
```diff
@@ -64,7 +64,7 @@ end
 platform :android do
 
   before_all do
-    apk_file_path = Dir["#{ENV["CIRCLE_ARTIFACTS"]}/*.apk"].first
+    apk_file_path = Dir["/home/circleci/code/cordova/platforms/android/app/build/outputs/apk/*/*.apk"].first
```
Comment (patrixr) [inline fastlane/Fastfile]: is the `CIRCLE_ARTIFACTS` variable gone ? 😱

Comment (patrixr) [reply fastlane/Fastfile]: oh sorry I missed that part of the description
Can't we replace it in the env instead of the Fastfile

**Review by patrixr — APPROVED** [review]
looks good, minuscule comments added

File: fastlane/Fastfile (hunk 2 above)
Comment (steveyken) [reply fastlane/Fastfile]: yes better to update in CIRCLE CI ENV

Comment (namrataukirde) [reply fastlane/Fastfile]: We can add our own environment variable for this so we can move it from fastfile. :)

**Review by steveyken — APPROVED** [review]
Thanks! Good job :)

**Pull request merged**